src/renderer/components/dock/dock.scss

```diff
@@ -26,7 +26,7 @@
       right: 0;
       left: 0;
       bottom: 0;
-      z-index: 100;
+      z-index: var(--z-index-base);
```

Comment:
Could we avoid using base as keep it since defaults (e.g. z-index == 0) where it needs?